### PR TITLE
Dartino mark sweep h naming convention

### PR DESCRIPTION
**Pull request by erikcorry**

### CODE REVIEW

File: src/third_party/dartino/mark_sweep.h [hunk 1]
```diff
@@ -90,83 +90,83 @@ class FreeList {
...
       }
       previous = current;
       current = reinterpret_cast<FreeListChunk*>(current->next_chunk());
     }
 
-    return NULL;
+    return null;
   }
 
   void Clear() {
```
Comment (kasperl) [inline src/third_party/dartino/mark_sweep.h:158]: clear?

File: src/third_party/dartino/mark_sweep.h [hunk 2]
```diff
@@ -90,83 +90,83 @@ class FreeList {
...
-    for (int i = 0; i < kNumberOfBuckets; i++) {
-      buckets_[i] = NULL;
+    for (int i = 0; i < NUMBER_OF_BUCKETS; i++) {
+      buckets_[i] = null;
     }
   }
 
   void Merge(FreeList* other) {
```
Comment (kasperl) [inline src/third_party/dartino/mark_sweep.h:164]: merge?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: src/third_party/dartino/mark_sweep.h [hunk 3]
```diff
@@ -90,83 +90,83 @@ class FreeList {
 #if defined(_MSC_VER)
   // Work around Visual Studo 2013 bug 802058
```
Comment (floitsch) [inline src/third_party/dartino/mark_sweep.h:91]: Consider removing this, if the bug is old enough.
Also, we currently don't even compile with visual studio code as the interpreter label-jumping is not supported by its compiler.

If you keep it, fix the typo in "Studo".

**Review by floitsch — COMMENTED** [review]
LGTM.

**Pull request merged**